ibkadeeko/Politico
# [feature #163550709] User can View Offices

#### What does this PR do?
Added a page where a user can view all offices
A user can follow a link on a particular office and see the candidates for that office
A user can vote for their preferred candidate on a particular office page
Added image upload to profile page
Refactored admin page
added phone number to registration
#### Description of Task to be completed?
A user can view Political offices and vote for their preferred Candidate
#### How should this be manually tested?
Navigate to the /office.html page, you would see a list of political offices, click the vote link on the president office to reveal the candidates and cast vote
#### Any background context you want to provide?
Implementing feedback
#### What are the relevant pivotal tracker stories?
#163550709
